### ISSUE TITLE
Point to enry and github/linguist

### ISSUE THREAD

**Issue by smola** [bblfsh/documentation#191]
From @ajnavarro yesterday ([more](https://github.com/ajnavarro/dupes/blob/master/empathy-log.md)):

> * I have the necessity of know the language of the file before hand to use bblfsh
>   * _talk about enry on bblfsh documentation to fill that necessity_

Somewhere in the documentation we should clarify that when we choose language keys, we use github/linguist as reference ([languages.yml](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)) and both enry in Go and linguist in Ruby will do the job if language detection is needed before passing to Babelfish.

**Comment by dennwc** [comment]
This is actually not true that you need to know the language to parse files. Bblfshd will autodetect it if it's empty. Need to clarify this.

**Comment by smola** [comment]
@dennwc Sure, but when you specify a language, we use language names as defined in enry.

**Comment by bzz** [comment]
Could somebody please help me understand, what are the action items here now?  

If that is to mention the language auto-detection + usage of enry/linguist language names on client API level - would https://docs.sourced.tech/babelfish/using-babelfish/clients be a good place for adding a new section on language detection?

**Comment by bzz** [comment]
@smola I see! But could you please help me understand how is that different from what we have in https://docs.sourced.tech/babelfish/using-babelfish/babelfish-protocol#parserequest ?

> he client must provide the code in the content field, the programming language in the language field (which must be one of the languages currently supported) or empty to enable auto-detection) and the filename field with the name of the file containing the source code.

From what I can see, it's:
 - the link to Enry is missing
 - field Key from https://docs.sourced.tech/babelfish/languages is not mentioned

If that is all that's needed - will be happy to add it asap.

**Comment by smola** [comment]
@bzz Yeah, a mention to the key would be enough I guess. Also, mention what's the behaviour regarding case sensitiveness (or lack of it).
